README.md

```diff
@@ -30,165 +32,471 @@ The two guiding principles are:
 1.  Hops scales with your requirements, from an easy start up to large scale applications with many teams
 2.  Hops fits a broad need by being modular and extensible, it comes with reasonable defaults, but allows you to customize, configure, and extend almost everything when needed
 
+These are the main features:
+
+- Server-side rendering + Universal rendering (the same code runs in both the server and the client)
+- Universal HMR (both the client and the server use webpack's HMR to update whenever you make edits)
+- Universal environment variable support at runtime
+- ES2018+ + JSX support (All new language features are supported via babel-preset-env)
+- Bundle Splitting (Universal bundle splitting with server-side rendering support)
+- Curated list of presets (for data fetching, styling, deployment, etc)
+- Sane defaults + Fully customizable
+
+## Table of contents
+
+- [System requirements](#system-requirements)
+- [Quick start](#quick-start)
+- [Walk-through](#walk-through)
+  - [Entry files and `hops`'s `render()` function](#entry-files-and-hopss-render-function)
+  - [Routing and `hops`'s `<Miss />` component](#routing-and-hopss-miss--component)
+  - [Styling Hops applications](#styling-hops-applications)
+    - [Styling with `hops-postcss`](#styling-with-hops-postcss)
+    - [Styling with `hops-styled-components`](#styling-with-hops-styled-components)
+  - [Code-splitting and `hops`s `Import()` function](#code-splitting-and-hopss-import-function)
+  - [CLI / npm scripts](#cli--npm-scripts)
+- [Configuration](#configuration)
+  - [Settings](#settings)
+    - [Default settings](#default-settings)
+    - [Placeholders](#placeholders)
+    - [Environment variables](#environment-variables)
+  - [Options](#options)
+- [Presets](#presets)
+  - [Installing presets](#installing-presets)
+  - [Activating presets](#activating-presets)
+  - [Available presets](#available-presets)
+    - [hops-redux](#hops-redux)
+    - [hops-graphql](#hops-graphql)
+    - [hops-postcss](#hops-postcss)
+    - [hops-styled-components](#hops-styled-components)
+    - [hops-typescript](#hops-typescript)
+    - [hops-pwa](#hops-pwa)
+    - [hops-development-proxy](#hops-development-proxy)
+    - [hops-lambda](#hops-lambda)
+- [Mixin (customizing behaviour)](#mixin-customizing-behaviour)
+- [Contributing](#contributing)
+- [Thanks](#thanks)
+
+## System requirements
+
+Hops is built on modern technologies and therefore needs at least Node.js _v8.10_ or higher.
+
+You can use either [`npx`]() (which is included in npm v5.2+) or [`npm init`]() (with npm v6+) or [`yarn create`]() to create a Hops application and run Hops CLI commands.
+
 ## Quick start
 
-Install the `hops` CLI globally:
+To create a new Hops application run the following in a terminal:
+
+```shell
+npx create-hops-app my-hops-app
```

Comment:
Do we want to mention here how this works with yarn? I.e. add 
`# OR yarn create hops-app my-hops-app`

Reply:
No, we decided against that. We are using npm and npx in all examples, because npx is available with npm 5.2+ and Node.js 8.10.0 (our minimum required version) ships with npm 5.6 (https://nodejs.org/en/download/releases/). People who use yarn, will probably know how to substitute these commands with yarn.
Also: In the "system requirements" section I also mention `yarn create` / `npm init` and npx.